sparsecca/_multicca_pmd.py

```diff
@@ -66,14 +73,14 @@ def multicca(datasets, penalties, niter=25, K=1, standardize=True, mimic_R=True)
 
     ws = []
     for idx in range(len(datasets)):
-        ws.append(svd(datasets[idx])[2][0:K].T)
+        ws.append(np.linalg.svd(datasets[idx])[2][0:K].T)
```

Comment:
@Moomboh could you comment here on this change?

Reply:
Must admit, that I probably left this over from debugging and don't remember exactly why. [NumPy seems to be faster though](https://stackoverflow.com/a/58137088) and this step is compute intensive.

Reply:
The SO actually says scipy is faster (presumably since it's built on fortran) so I would prefer reverting to scipy here (with a comment that perhaps someone who is annoyed at the runtime can change to jax).